docs/help/git_webhooks.md

```diff
@@ -0,0 +1,30 @@
+## Webhooks
+
+1. Generate a SSL certificate for the REST API. TODO: figure out the right
+   way to do that.
+
+Using acme.sh on the host already, I did TODO
+
+```
+$ACMESH --key-file $BITBOTDIR/ssl.key --cert-file $BITBOTDIR/ssl.cert --reloadcmd "$SYSTEMCTLRESTART bitbot"
+chmod -R 700 $BITBOTDIR/ssl.key
+chown -R bitbot:bitbot $BITBOTDIR
+```
+
+2. Enable the REST API in bot.conf by adjusting
+
+```
+# key/cert used for REST API
+tls-api-key              = /path/to/just/generated/key
+tls-api-certificate      = /path/to/just/generated/cert
+api-port                 = 5000
+```
+
+you may change the API port if you wish.
+
+3. restart the bot
+
+4. run `!config bot rest-api on`.
```

Comment:
The guide issue told me to `!botset rest-api on` which the bot doesn't respond to at all. I got a response with lucky guess `!config bot rest-api on`, but I am unable to telnet into port 5000 or nmap it (it appears as closed) and the bot only told me `[Config] Config 'rest-api' set to True`, so I assumed that only the proper webhooks are going to work and continued ahead.

Reply:
Oh, I see